proxy/gating.yaml

```diff
@@ -9,7 +9,7 @@ decision_contexts:
 rules:
   # Rules are in format <namespace>.<type>.<category>
   # TODO - need proxy test suite
```

Comment:
We should probably remove the TODO and the other commented out lines.

Reply:
The other commented lines would apply if we support mode architectures in the future